pages/_app.tsx

```diff
@@ -0,0 +1,15 @@
+import App from 'next/app';
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import theme from 'styles/theme';
+
+export default class MyApp extends App {
+  render() {
+    const { Component, pageProps } = this.props;
+    return (
+      <ThemeProvider theme={theme}>
```

Comment:
are we going to provide dark theme in the future?

Reply:
If so I'd recommend with this media query `prefers-color-scheme:dark {}`

Reply:
Browser coverage is very less for `prefers-color-scheme:dark {}`
![image](https://user-images.githubusercontent.com/8705386/67427820-872ccc80-f5fa-11e9-9e6a-880c3a9004e2.png)

Reply:
It's something we can look at - one of the advantages of using a global theme is that it's one source of truth for the entire application and whitelabelling and re-theming are a walk in the park

Reply:
Wow, in that case, I'd favor something with more adoption then; like css vars you could change with a toggle or something. I'm sure ThemeProvider is fine too.